core/runtime.rs

```diff
@@ -476,35 +479,44 @@ impl JsRuntime {
         self.register_op(name, macroware(name, opfn));
       }
     }
-    // Sync ops cache
-    self.sync_ops_cache();
     // Restore extensions
     self.extensions = extensions;
 
     Ok(())
   }
 
-  /// Grabs a reference to core.js' opresolve
-  fn init_recv_cb(&mut self) {
-    let scope = &mut self.handle_scope();
-
-    // Get Deno.core.opresolve
-    let code = v8::String::new(scope, "Deno.core.opresolve").unwrap();
+  /// Grab a Global handle to a function returned by the given expression
+  fn grab_fn(
+    scope: &mut v8::HandleScope,
+    code: &str,
+  ) -> v8::Global<v8::Function> {
```

Comment:
Does this have to be specific to functions? One of the possibilities for implementing #12067 is to define `Deno.serialize` and `Deno.deserialize` symbols, which would have to be accessible from `bindings.rs`. Could this function be generic over `T where for<'s> v8::Local<'s, T>: TryFrom<v8::Local<'s, v8::Value>>`?

Reply:
Sure, there's this could be made generic, but I would rather do that in a separate PR or one implementing #12067 since there's no immediate need in this PR for that helper to be generic

Reply:
SGTM